package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "npm run build",
```

Comment:
Recursion!

Reply:
What do you mean, that I was making it run.... @dahlbyk